src/main/java/com/faforever/client/map/MapCardController.java

```diff
@@ -87,8 +97,16 @@ public void setMap(MapBean map) {
     sizeLabel.setText(i18n.get("mapPreview.size", size.getWidthInKm(), size.getHeightInKm()));
     maxPlayersLabel.setText(i18n.number(map.getPlayers()));
 
-    ObservableList<MapBean> installedMaps = mapService.getInstalledMaps();
-    JavaFxUtil.addListener(installedMaps, new WeakListChangeListener<>(installedMapsChangeListener));
+    if (mapService.isOfficialMap(map.getFolderName())) {
+      installButton.setDisable(false);
+      uninstallButton.setDisable(false);
```

Comment:
what for?

Reply:
ha meant to disable it but got the logic wrong. Meant to be disabled so that if anything else in the client tries to manually fire it it wont do anything

Reply:
Dont thin that invisible buttons can be fired but well ok

Reply:
ah I was probably being over cautious